src/reference/config/catalogs/catalog-aws-glue.md

```diff
@@ -0,0 +1,64 @@
+---
+shortTitle: aws-glue
+description: Zilla runtime aws-glue catalog
+icon: aky-zilla-plus
+category:
+  - Catalog
+---
+
+# aws-glue Catalog
+
+[Available in <ZillaPlus/>](https://www.aklivity.io/products/zilla-plus)
+{.zilla-plus-badge .hint-container .info}
+
+Zilla runtime aws-glue catalog
+
+```yaml {2}
+catalog:
+  type: aws-glue
+  options:
+    registry: zilla
+    max-age: 30
+    compression: zlib
+```
+
+## Summary
+
+Defines a catalog with a schema pulled from aws glue to enforce validation.
+
+## Configuration
+
+:::: note Properties
+
+- [options](#options)
+  - [options.registry\*](#options-registry)
+  - [options.max-age](#options-max-age)
+  - [options.compression](#options-compression)
+
+::: right
+\* required
+:::
+
+::::
+
+### options
+
+> `object`
+
+#### options.registry\*
+
+> `string`
+
+AWS Glue Registry name to access schemas.
+
+#### options.max-age
+
+> `number` | Default: `"300"`
+
+Configure the Time-To-Live for the schema information retrieved against latest version. Defaults to 5 minutes. 
+
+#### options.compression
```

Comment:
```suggestion
#### options.model-compression
```

I think the `model-` prefix adds just enough information to tie it to the model-view implementations. It also reads as a global setting for all models, which we can clarify in the property description. 

@jfallows @aDaemonThread

Reply:
@vordimous Let's go with what we have for now, as this is already released in Zilla Plus and we need to get the docs out to let folks be successful with it.

We can consider an evolution in this syntax for a future Zilla Plus release.